crane/container.go

```diff
@@ -69,15 +69,15 @@ type RunParameters struct {
 	RawCapAdd       []string    `json:"cap-add" yaml:"cap-add"`
 	RawCapDrop      []string    `json:"cap-drop" yaml:"cap-drop"`
 	RawCgroupParent string      `json:"cgroup-parent" yaml:"cgroup-parent"`
-	CpuPeriod       int         `json:"cpu-period" yaml:"cpu-period"`
```

Comment:
I'm not entirely sure, but I think I read at some point that gaoling prefers camel case here. I definitely think that it is easier to read then the alternativ versions. Mind removing all of those changes?